Frameworks/UIKit/UIFont.mm

```diff
@@ -362,99 +65,35 @@ + (UIFont*)preferredFontForTextStyle:(NSString*)style {
     return StubReturn();
 }
 
+/**
+ @Status Stub
+*/
 + (UIFont*)fontWithData:(NSData*)data {
```

Comment:
May be we should have a TODO for such things that we can revisit and have a work item or bug to fix

---
In reply to: [78118792](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78118792) [](ancestors = 78118792)